src/github.com/getlantern/launcher/launcher_windows.go

```diff
@@ -8,13 +8,16 @@ import (
 	"github.com/getlantern/golog"
 )
 
+const (
+	runDir = `Software\Microsoft\Windows\CurrentVersion\Run`
```

Comment:
Yeah, that's actually identical to what I had originally =P @myleshorton recommended putting it inside the only function it's used in. I don't think it makes a big difference, but I prefer making it clear its a constant too.